llama_index/agent/openai_agent.py

```diff
@@ -142,12 +134,14 @@ def __init__(
         self,
         llm: OpenAI,
         chat_history: List[ChatMessage],
+        prefix_messages: Optional[List[ChatMessage]],
         verbose: bool = False,
         max_function_calls: int = DEFAULT_MAX_FUNCTION_CALLS,
         callback_manager: Optional[CallbackManager] = None,
     ) -> None:
         self._llm = llm
         self._chat_history = chat_history
+        self._prefix_messages = prefix_messages
```

Comment:
This isn't actually used in the chat functions?

Reply:
Might want to add it to `init_chat` for now? Will need some extra thought once we have memory modules

Reply:
ooops, missing commits here.